src/event.py

```diff
@@ -26,7 +26,7 @@ def parse_location(event_json):
         )
 
     if event_json["venue"]["lat"] is not None and event_json["venue"]["lat"]:
```

Comment:
```suggestion
    if event_json["venue"]["lat"] is not None and event_json["venue"]["lon"] is not None:
```

Would this make more sense just for completeness?